gensrc/java/com/starrocks/udf/UDFClassAnalyzer.java

```diff
@@ -0,0 +1,71 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+package com.starrocks.udf;
+
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.lang.reflect.Type;
+import java.util.HashMap;
+import java.util.Map;
+
+public class UDFClassAnalyzer {
+    static Map<String, String> anlyMap = new HashMap<>();
+
+    static {
+        anlyMap.put("boolean", "Z");
+        anlyMap.put("byte", "B");
+        anlyMap.put("char", "C");
+        anlyMap.put("short", "S");
+        anlyMap.put("int", "I");
+        anlyMap.put("long", "J");
+        anlyMap.put("float", "F");
+        anlyMap.put("double", "D");
+        anlyMap.put("void", "V");
+    }
+
+    private static String getSign(String typeName) {
+        String prefix = "";
+        if (typeName.contains("[]")) {
+            prefix = "[";
+            typeName = typeName.replace("[]", "");
+        }
+        String signStr = anlyMap.get(typeName);
+        if (signStr != null) {
+            return prefix + signStr;
+        } else {
+            return prefix + "L" + typeName.replace('.', '/') + ";";
+        }
+    }
+
+    public static boolean hasMemberMethod(String methodName, Class clazz) {
+        for (Method declaredMethod : clazz.getDeclaredMethods()) {
+            if (declaredMethod.getName().equals(methodName)) {
+                if (Modifier.isStatic(declaredMethod.getModifiers())) {
+                    return false;
+                }
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public static String getSign(String methodName, Class clazz) throws NoSuchMethodException {
+        for (Method declaredMethod : clazz.getDeclaredMethods()) {
```

Comment:
Don't we already depends on assumption that only one method named `evaluate` is supported? Do we need to handle signature?